manim/mobject/vector_field.py

```diff
@@ -25,7 +27,7 @@
 from ..animation.composition import AnimationGroup, Succession
 from ..animation.creation import Create
 from ..animation.indication import ShowPassingFlash
-from ..constants import OUT, RIGHT, UP
+from ..constants import ORIGIN, OUT, RIGHT, UP
```

Comment:
`ORIGIN` does not seem to be used in the diff, can it be removed from the imports?